books.php

```diff
@@ -0,0 +1,121 @@
+<?php 
+   namespace db;
+   class book
+   {
+      public $id;
+      public $title;
+      public $author;
+      public $category;
+      public $myDate;
+      public $qty;
+      public $available;
+      public $pdo;
+      public function __construct()
+      {
+         $this->pdo= new PDO('mysql:host=localhost;port=3307;dbname=bookhive', 'userid', 'password');
```

Comment:
use the creds file to get the credentials.

Reply:
And if possible can I have a function which does kind of -> https://github.com/JayKandari/bookhive/issues/8#issuecomment-696330392